samples/net/sockets/civetweb/prj.conf

```diff
@@ -13,12 +13,12 @@ CONFIG_NET_IPV4=y
 # CONFIG_NET_IPV6 is not set
```

Comment:
In commit "samples: civetweb: fixes #....", could you places the "Fixes #21179" to the body of the commit message, and write something more descriptive to the subject. Something like
```
samples: net: civetweb: Reduce RAM usage

Adjust configuration so that Civetweb uses less memory.

Fixes #21179

Signed-off-by: <insert your name and mail here>
```

Reply:
will be updated with next push